test/test_plugin_helper.rb

```diff
@@ -6,3 +6,21 @@
 # Add plugin to FactoryGirl's paths
 FactoryGirl.definition_file_paths << File.join(File.dirname(__FILE__), 'factories')
 FactoryGirl.reload
+
+# Foreman's setup doesn't handle cleaning up for Minitest::Spec
+DatabaseCleaner.strategy = :transaction
+
+class Minitest::Spec
+  class << self
+    alias_method :context, :describe
+  end
+
+  before :each do
+    DatabaseCleaner.start
+    Setting::RemoteExecution.load_defaults
+  end
+
+  after :each do
+    DatabaseCleaner.clean
```

Comment:
hrmmmm, this seems kinda odd.  I wouldn't think using a transaction strategy would be too slow as it would be doing what the test suites are doing for other projects.  I don't just quite understand why its needed.